### ISSUE TITLE
Translation of 'Some As are Bs' sentences

### ISSUE THREAD

**Issue by mereolog** [Yale-LILY/FOLIO#2]
I think that your translation of the sentences of the form 'Some As are Bs' is incorrect, e.g.:

Some musicians love music. > ∃x (Musician(x) **→** Love(x, music))
Some fish may sting. > ∃x ∃y (Fish(x) **→** Sting(x,y))

These should be translated with the help of the conjunction and not implication, e.g.

Some musicians love music. > ∃x (Musician(x) **∧** Love(x, music))
Some fish may sting. > ∃x ∃y (Fish(x) **∧** Sting(x,y))

Consult, for example, https://cse.buffalo.edu/~rapaport/191/S09/transtip-pnllogic.html#plsinglymonadic.

The reason is that the "implication" formal versions are semantically "too weak" - they are satisfied also when there are no musicians, no fish, etc.

**Comment by SophiaHan6** [comment]
Yes, we are aware that conjunction should be used with the existential quantifier. We corrected most of the errors during the review stage where implication was used with the existential quantifier, but there are some instances in the current version of the dataset that still has this error. We will correct these errors and release a new version very soon!
